services/ui-src/src/components/pages/ReviewSubmit/McparReviewSubmitPage.tsx

```diff
@@ -111,7 +115,11 @@ const ReadyToSubmit = ({
         </Box>
       </Box>
       <Flex sx={sx.submitContainer}>
-        <Button type="submit" onClick={onOpen as MouseEventHandler}>
+        <Button
+          type="submit"
+          onClick={onOpen as MouseEventHandler}
+          isDisabled={!isPermittedToSubmit}
```

Comment:
Note: It's very weird to me that Chakra UI has the prop be `isDisabled` when the rest of the internet just uses `disabled`. Either will work here. 🤷